testnets/sunrisetestnet/assetlist.json

```diff
@@ -0,0 +1,70 @@
+{
+  "$schema": "../../assetlist.schema.json",
+  "chain_name": "sunrise",
```

Comment:
this chain_name should be 'sunrisetestnet'

Reply:
fixed. thanks